cloudbuild/Dockerfile

```diff
@@ -11,6 +11,6 @@ COPY --chown=ia-tests:ia-tests . /init-actions
 # https://docs.bazel.build/versions/master/install-ubuntu.html
 RUN echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
 RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
-RUN apt-get update && apt-get install -y openjdk-8-jdk python3-setuptools bazel
+RUN apt-get update && apt-get install -y openjdk-8-jdk python3-setuptools bazel gettext-base
```

Comment:
that seems fine.  I don't see it being used below, but perhaps the kubectl apply depends on this for translation.  Can you share why we included this package?